server.py

```diff
@@ -61,22 +79,18 @@ def create_ffmpeg_stream(video_path:str, video_type:State, loop=False):
         stderr=subprocess.DEVNULL
     )
     process_dict[video_type] = process.pid
-    MetricsHandler.streams_count.labels(video_type=video_type.value).inc(amount=1)
```

Comment:
should we keep this

Reply:
Yes, that was my fault. I might have accidentally deleted that when working on the server.py file, but I believe I would not crash our software if we had it or not, but anyway I'll still put it back.